scripts/globals/effects/bust.lua

```diff
@@ -19,24 +15,16 @@ function onEffectGain(target,effect)
         elseif (effect:getSubType() == dsp.mod.ATTP) then
             target:addMod(dsp.mod.RATTP, -effect:getPower());
         elseif (effect:getSubType() == MOD_PET_MACC) then
-            target:addMod(MOD_PET_MATT, -effect:getPower());
+            target:addMod(dsp.mod.PET_MATT, -effect:getPower());
```

Comment:
PET_MATT does not exist in status.lua or modifier.h

Reply:
ha ha..hahaha.. HAHAHAHAHAHA *dies* :skull: :ghost:

Reply:
I missed the conditional above it too `MOD_PET_MACC`

Reply:
I'm confident that the search/replace program caught all the cases, so any mods (or other stuff I tabled) that remain unconverted are either typos, not yet implemented, missing from status.lua, or wacky in some other manner. :)

Reply:
ages ago the project nuke mod_pet_whatever and moved to using addModPet(normal modifier here) instead. so the conditional here doesn't even make sense unless that was defined some other place and shouldn't have been prefixed `mod_`

I've straight up commented the block out. someone else can sort it out, at least it doesn't crash now.